Detectors/ITSMFT/ITS/postprocessing/studies/src/MCKinematicReaderSpec.cxx

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include <ITSStudies/MCKinematicReaderSpec.h>
+#include <Framework/ControlService.h>
+
+namespace o2
+{
+namespace steer
+{
+using namespace o2::framework;
+
+void KinematicReader::init(o2::framework::InitContext& ic)
```

Comment:
Hi @mconcas : 

As you know, there is an existing DPL device that is 99% the same. So in principle I would want to avoid duplicating code and rather add new features there.